erpnext/translations/bn.csv

```diff
@@ -2297,7 +2297,11 @@ DocType: Timesheet Detail,Expected Hrs,প্রত্যাশিত ঘন্
 apps/erpnext/erpnext/config/non_profit.py +28,Memebership Details,মেমরিরশিপ বিস্তারিত
 DocType: Leave Block List,Block Holidays on important days.,গুরুত্বপূর্ণ দিন অবরোধ ছুটির দিন.
 apps/erpnext/erpnext/healthcare/doctype/lab_test/lab_test.js +194,Please input all required Result Value(s),সব প্রয়োজনীয় ফলাফল মান (গুলি) ইনপুট করুন
+<<<<<<< HEAD
```

Comment:
please fix merge conflicts.

Reply:
Hey you can go ahead now, conflicts have been resolved